ci-operator/step-registry/baremetal/lab/ipi/conf/ntp/baremetal-lab-ipi-conf-ntp-ref.yaml

```diff
@@ -0,0 +1,26 @@
+ref:
+  as: baremetal-lab-ipi-conf-ntp
+  from_image:
+    namespace: ci
+    name: "baremetal-qe-base"
+    tag: latest
+  commands: baremetal-lab-ipi-conf-ntp-commands.sh
+  grace_period: 600
+  resources:
+    requests:
+      cpu: '1'
+      memory: 128Mi
+  env:
+    - name: INTERNAL_NET_CIDR
+      default: "192.168.80.0/22"
+      documentation: ""
+    - name: ipv6_enabled
+      default: "false"
+    - name: ipv4_enabled
+      default: "static"
```

Comment:
ipv[64]_enabled are currently booleans. I would only use them to distinguish whether to add an ipv4 or ipv6 NTP server for now.

Reply:
Well, as you're using the hostname, we don't even need to distinguish the cases.